docs/magidoc.mjs

```diff
@@ -67,8 +66,8 @@ const config = {
   },
   dev: {
     watch: [
-      path.join(__dirname, './pages'),
-      path.join(__dirname, './pages.mjs'),
+      fileURLToPath(new URL('./pages', import.meta.url)),
```

Comment:
Thanks for teaching me about this approach. I always found the way I was doing cumbersome, this is definitely better. I would really enjoy if node could add a standard function to get a relative path to the current location tho. Feels a bit weird to concatenate paths in the reverse order (pass the base path after the relative path).